scripts/xnat/t1_qa_functions.py

```diff
@@ -154,4 +157,7 @@ def process_phantom_session( interface, project, subject, session, label, force_
             run_phantom_qa( interface, project, subject, session, label, dicom_path )
```

Comment:
ah, sorry - github had collapsed the code making it look like it was more indented... nevermind